source/description.rst

```diff
@@ -34,6 +34,11 @@ the format in order to store types of data not supported by the
       {{ spec_format }}. For further details, see the {{ spec_format }} docs available here:
       http://{{ spec_format }}.readthedocs.io/en/latest/index.html
 
+.. note::
+
+    Each API for working with the {{ spec_format }} specification language may not yet provide
+    full support for all features of the specification language. Please consult the API documentation
+    for details.
```

Comment:
I think it would be useful in this case to have a list of limitations/restrictions in the HDMF docs somewhere that we can point to. The note by itself is a bit generic and may be confusing because it is not clear what restrictions this is referring. For the schema language docs this is fine, but it would be nice to at least have link to the limitations of the reference API

Reply:
Done